hathor/transaction/base_transaction.py

```diff
@@ -832,7 +832,8 @@ def get_metadata(self, *, force_reload: bool = False, use_storage: bool = True)
             #        which requires the use of a storage, this is a workaround that should be fixed, places where this
             #        happens include generating new mining blocks and some tests
             height = self.calculate_height() if self.storage else 0
-            metadata = TransactionMetadata(hash=self.hash, accumulated_weight=self.weight, height=height)
+            score = self.weight if self.is_genesis else 0
+            metadata = TransactionMetadata(hash=self.hash, accumulated_weight=self.weight, height=height, score=score)
```

Comment:
Why do we need this change?

Reply:
Basically to have a correct `accumulated_weight` metadata for freshly generated metadata of genesis transactions. This appeared on some tests because of the changes to the manager initialization. And honestly, I'm not completely sure how it was working before.

Reply:
Make sense. Thanks!